src/MultiOwnable.sol

```diff
@@ -49,6 +49,13 @@ contract MultiOwnable {
     /// @param index The targeted index for removal.
     error NoOwnerAtIndex(uint256 index);
 
+    /// @notice Thrown when `owner` argument does not match owner found at index.
+    ///         the provided owner.
```

Comment:
@stevieraykatz accidentally leave this?

Reply:
Agh, looks like an artifact from accepting suggestions in gh ui. Fixed